modules/core/js_modules/Hm_MessagesStore.js

```diff
@@ -164,7 +165,7 @@ class Hm_MessagesStore {
                     break;
                 case 'tag':
                     hook = "ajax_imap_tag_data";
-                    folder = getParam('tag_id');
+                    tag_id = getParam('tag_id');
```

Comment:
Please revert here.

Reply:
That is the tag_id why revert ?

Reply:
So, from our entire conversation last Saturday, it seems you didn’t grasp the key points. I recommend revisiting and carefully reviewing all the chats.